### ISSUE TITLE
GitSavvy views won't open if no tabs are present

### ISSUE THREAD

**Issue by alextegelid** [divmain/GitSavvy#128]
For example the status view won't open unless I open one or more files in tabs. Executing the "git: status" command outputs this in console

```
Traceback (most recent call last):
  File "/Applications/Sublime Text.app/Contents/MacOS/sublime_plugin.py", line 526, in run_
    return self.run()
  File "/Users/alex/Library/Application Support/Sublime Text 3/Packages/GitSavvy/core/interfaces/status.py", line 97, in run
    repo_path = self.repo_path
  File "/Users/alex/Library/Application Support/Sublime Text 3/Packages/GitSavvy/core/git_command.py", line 148, in repo_path
    return self._repo_path()
  File "/Users/alex/Library/Application Support/Sublime Text 3/Packages/GitSavvy/core/git_command.py", line 163, in _repo_path
    file_dir = os.path.dirname(file_path)
  File "./posixpath.py", line 151, in dirname
AttributeError: 'NoneType' object has no attribute 'rfind'
```

Might this have something to do with reusing gitsavvys tabs if present? Maybe that function needs to check if any tabs are open before trying to reuse them. I'm not familiar with ST plugin development, so this is just a guess ;)

**Comment by yyyc514** [comment]
Try PR #130

**Comment by alextegelid** [comment]
Thanks! I'll wait for @divmain to merge it.

**Comment by divmain** [comment]
@alextegelid, this should be fixed in master, along with #129.  Please let me know if you encounter any issues.

**Comment by alextegelid** [comment]
Thanks! I'll continue to keep an eye open for further issues. Thanks again for writing this
